### PR TITLE
Improved some phrases

### PR DESCRIPTION
**Pull request by RedBrumbler**
they were not entirely correct, or just sounded weird

### CODE REVIEW

**Review by dukedagmor — APPROVED** [review]

**Comment by dukedagmor** [comment]
As a dutch native speaker I can confirm this is better

File: ModAssistant/Localisation/nl.xaml [hunk 1]
```diff
@@ -189,19 +189,19 @@
...
-        Als dat allebei niet helpt, vraag om hulp in de 
-        <Span Foreground="Blue">#support</Span> channel in
+        Als dat allebei niet helpt, vraag om hulp in het 
+        <Span Foreground="Blue">#support</Span> kanaal in
         <Hyperlink NavigateUri="https://discord.gg/beatsabermods" local:HyperlinkExtensions.IsExternal="True">
             BSMG
         </Hyperlink>. (Engels)
     </Span>
-    <sys:String x:Key="Invalid:BoughtGame1">Als U een gepirateerde versie van het spel had maar nu het spel hebt gekocht</sys:String>
+    <sys:String x:Key="Invalid:BoughtGame1">Als U een illegaal gedownloade versie van het spel had maar nu het spel hebt gekocht</sys:String>
```
Comment (JensSchadron) [inline ModAssistant/Localisation/nl.xaml:204]: Missing comma between "had" and "maar"

Comment (ErisApps) [inline ModAssistant/Localisation/nl.xaml:204]: Missing comma between "had" and "maar"

Comment (ErisApps) [inline ModAssistant/Localisation/nl.xaml:204]:
```suggestion
    <sys:String x:Key="Invalid:BoughtGame1">Als U een illegaal gedownloade versie van het spel had, maar nu het spel heeft gekocht</sys:String>
```

**Pull request merged**